deploy/manifests/dev-v2/us-east-2/cluster/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../base/ingress-nginx
+  - kube-system
+  # - external-dns
```

Comment:
I see this is disabled. Is there plan to re-enable it later? 

It would be great to either remove all manifests related to this from the PR or re-enable it. See earlier comment re hosted zone

Reply:
@masih external dns has been disabled until I can complete this ticket.
https://github.com/filecoin-project/gitops-root/issues/196

dev-2 will get it's own zone. if the current plan is followed.

I am going to remove external dns to avoid confusion.